src/content/structured/components/badge/code.mdx

```diff
@@ -0,0 +1,755 @@
+---
+path: "/components/badge/code"
+
+date: "2023-02-03"
+
+title: "Badge"
+
+status: "BETA"
+
+subTitle: "Badges are attached to other components to provide an indication that something new is available, or that something has changed."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/badge"
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/badge" },
+    { title: "Code", path: "/components/badge/code" },
+    { title: "Accessibility", path: "/components/badge/accessibility" },
+  ]
+---
+
+import { IcButton, IcBadge, SlottedSVG, IcChip, IcTab, IcTabPanel, IcTabContext, IcTabGroup } from "@ukic/react";
+import { useRef } from "react";
+
+## Component demo
+
+export const snippetsDefault = [
+  {
+    language: "Web component",
+    snippet: `<ic-button variant="secondary" style="margin-right:16px">
+    <ic-badge text-label="1" slot="badge" variant="info"/>
+    Default
+  </ic-button>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcButton variant="secondary">
+    <IcBadge text-label="1" slot="badge" variant="info"/>
+    Default
+  </IcButton>`,
```

Comment:
```suggestion
  <IcBadge text-label="1" slot="badge" variant="info"/>
  Default
</IcButton>`,
```

Reply:
This snippet has now changed but should be aligned properly